Marlin/Marlin_main.cpp

```diff
@@ -10185,6 +10201,104 @@ inline void gcode_M502() {
 
 #endif // DUAL_NOZZLE_DUPLICATION_MODE
 
+#if ENABLED(FILAMENT_LOAD_UNLOAD_GCODES)
+
+  /**
+   * M701: Load filament
+   *
+   *  T[extruder] - Optional extruder number. Current extruder if omitted.
+   *  L[distance] - Extrude distance for insertion (positive value) (manual reload)
+   *
+   *  Default values are used for omitted arguments.
+   *
+   */
+  inline void gcode_M701() {
+
+    if (get_target_extruder_from_command(200)) return;
+
+    // Load filament
+    const float load_length = parser.seen('L') ? parser.value_axis_units(E_AXIS) : 0
+      #ifdef FILAMENT_CHANGE_LOAD_LENGTH
+        + FILAMENT_CHANGE_LOAD_LENGTH
+      #endif
+    ;
+
+    #if ENABLED(MK2_MULTIPLEXER)
+      if (target_extruder != active_extruder) select_multiplexed_stepper(target_extruder);
+    #endif
+
+    load_filament(target_extruder, load_length, ADVANCED_PAUSE_EXTRUDE_LENGTH, FILAMENT_CHANGE_NUMBER_OF_ALERT_BEEPS,
+                  true, false, ADVANCED_PAUSE_MODE_LOAD_FILAMENT);
+
+    #if ENABLED(MK2_MULTIPLEXER)
+      if (target_extruder != active_extruder) select_multiplexed_stepper(active_extruder);
+    #endif
+  }
+
+  /**
+   * M702: Unload filament
+   *
+   *  T[extruder] - Optional extruder number. If omitted, current extruder
+   *                (or ALL extruders with FILAMENT_UNLOAD_ALL_EXTRUDERS).
+   *  Z[distance] - Move the Z axis by this distance
+   *  U[distance] - Retract distance for removal (manual reload)
+   *
+   *  Default values are used for omitted arguments.
+   *
+   */
+  inline void gcode_M702() {
+    point_t park_point = NOZZLE_PARK_POINT;
+
+    if (get_target_extruder_from_command(702)) return;
+
+    // Z axis lift
+    if (parser.seenval('Z')) park_point.z = parser.linearval('Z');
+
+    // Unload length
+    const float unload_length = -FABS(parser.seen('U') ? parser.value_axis_units(E_AXIS) : 0
+      #if FILAMENT_CHANGE_UNLOAD_LENGTH
+        + (FILAMENT_CHANGE_UNLOAD_LENGTH)
+      #endif
+    );
+
+    // Lift Z axis
+    if (park_point.z > 0)
+      do_blocking_move_to_z(min(current_position[Z_AXIS] + park_point.z, Z_MAX_POS), NOZZLE_PARK_Z_FEEDRATE);
+
+    // Unload filament
+    #if ENABLED(FILAMENT_UNLOAD_ALL_EXTRUDERS)
+      if (!parser.seenval('T')) {
+        HOTEND_LOOP() {
+          #if ENABLED(MK2_MULTIPLEXER)
+            select_multiplexed_stepper(e);
+          #endif
+          unload_filament(e, unload_length, true, ADVANCED_PAUSE_MODE_UNLOAD_FILAMENT);
+        }
+        #if ENABLED(MK2_MULTIPLEXER)
+          select_multiplexed_stepper(active_extruder);
+        #endif
```

Comment:
@tcm0116 I did catch a typo/bug here. The `if` statement didn't have a curly brace, so this multiplexer line wasn't inside the if block. The hanging `else` wasn't caught probably because `.travis.yml` probably doesn't test all this yet.

Reply:
…but now it does…